webapp/portlet/src/main/webapp/vue-apps/popover/components/PopoverMenu.vue

```diff
@@ -119,6 +120,12 @@ export default {
     document.addEventListener('drawerClosed', () => this.closePopover(true));
     document.addEventListener('modalOpened', () => this.closePopover(true));
     document.addEventListener('modalClosed', () => this.closePopover(true));
+    document.addEventListener('popover-identity-hide', () => this.closePopover(true));
```

Comment:
What is this event for ?

Reply:
for the focusout event we need to hide the popover

Reply:
https://github.com/Meeds-io/social/pull/3447/files#diff-49e191f2a3cc7bb6bf8168652aa3d5cc741da346cf329456db20bb9e8bc9aa5bR32